src/hooks/diary/queries/useFetchRecallQuiz.js

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from "@tanstack/react-query";
+import DiaryRecallController from "../../../apis/diary.recall.controller";
+const useFetchDiaryRecallQuiz = (diaryId) => {
```

Comment:
```suggestion
const useFetchDiaryRecallQuiz = () => {
  const { diary } = useFetchDiary();
  const { diaryId } = diary;
```

diaryId 지금 코드에서 가져올수 있으니까 수정해 주세요